File: code/char/parse-tsv.pl
#!/usr/bin/env perl

use Smart::Comments;
use strict;

my @tbl;
while (<>) {
    chomp;
    # フィールドを分割する
    my @cols = split /\t/, $_;
    # \t および \t を置換する
    @cols = map { s/\\t/\t/g; s/\\n/\n/g; $_; } @cols;
    push @tbl, [@cols];
}
### @tbl

File: code/char/parse-tsv.py
#!/usr/bin/env python3
import sys

file = sys.argv[1]
with open(file) as f:
    tbl = []
    for line in f:
        line = line.rstrip()
        cols = line.split('\t')
        cols = [f.replace('\\t', '\t').replace('\\n', '\n') for f in cols]
        tbl.append(cols)
    print(tbl)
